apps/api/src/app/controllers/utilities/buildIndexPathForResourceType.ts

```diff
@@ -3,7 +3,7 @@ import { RESOURCES_ROUTE_PREFIX } from '../resources/constants';
 
 // For irregular plural forms or forms that lead to awkward grammar
 const exceptions: Partial<Record<ResourceType, string>> = {
-    [ResourceType.transcribedAudio]: 'transcribedAudioItems',
+    [ResourceType.audioItem]: 'transcribedAudioItems',
```

Comment:
can't we remove this entry now?

Reply:
I added a comment. We can update this next time we hit a legit irregular plural. It's a nice placeholder if nothing else.